src/main/java/com/targomo/client/api/util/POJOUtil.java

```diff
@@ -12,8 +13,11 @@
      * @return the pretty print String
      */
     public static String prettyPrintPOJO(Object plainOldJavaObject) {
-        try (CharBuf buf = CharBuf.createCharBuf()) {
-            return buf.prettyPrintBean(plainOldJavaObject).toString();
+        ObjectMapper mapper = new ObjectMapper();
+        try {
+            return mapper.writerWithDefaultPrettyPrinter().writeValueAsString(plainOldJavaObject);
+        } catch (JsonProcessingException e) {
+            return "";
```

Comment:
probably better to pass on the exception, no?

Reply:
Usually this method is called when we have an exception (to print all the fields of pojo). So passing the exception while printing the object state on the first exception didn't seem fine to me. What do you think?

Reply:
I think you won't know what went wrong if an exception happens and you just return "". Maybe at least return the error message of the exception so the user can work with that

Reply:
oki, done